pkg/daemon/osrelease.go

```diff
@@ -17,21 +17,27 @@ type OperatingSystem struct {
 	VersionID string
 }
 
-// IsRHCOS is true if the OS is RHEL CoreOS
-func (os OperatingSystem) IsRHCOS() bool {
-	return os.ID == "rhcos"
+// IsEL is true if the OS is an Enterprise Linux variant,
+// i.e. RHEL CoreOS (RHCOS) or CentOS Stream CoreOS (SCOS)
+func (os OperatingSystem) IsEL() bool {
+	return os.ID == "rhcos" || os.ID == "scos"
```

Comment:
This is OK as is but...the better way to do this I think is to use the `ID_LIKE` field - it's intended for exactly this use case.  IOW we'd check whether `strings.Contains(os.ID_LIKE, "rhel")` (or to do it a bit more precisely, split `ID_LIKE` into a proper whitespace separated array of strings).

In particular, this would make things Just Work on non-Stream CentOS, Alma, etc.

Reply:
hm, wouldn't that only be safe in combination with `VARIANT_ID == "coreos"`? (which RHCOS8 doesn't specify)

Reply:
The func isn't particularly well named, it probably should've been `IsELCOS()`